### PR TITLE
Email unsubscribe tracking

### PR DESCRIPTION
**Pull request by kangguru**
implements the `unsubscribe` endpoint for tracking campaign unsubscribes

http://documentation.emarsys.com/resource/developers/endpoints/email/unsubscribe-from-launch/

see #48

### CODE REVIEW

**Comment by kangguru** [comment]
bump :)

anything we can help with? 

btw: happy xmas 🎄

File: spec/emarsys/data_objects/email_spec.rb [hunk 1]
```diff
@@ -83,6 +83,16 @@
     end
   end
 
+  describe ".unsubscribe" do
+    it "requests an email test sending with custom recipient list" do
```
Comment (mungler) [inline spec/emarsys/data_objects/email_spec.rb]: Is this copy-pasta?

Comment (kangguru) [reply spec/emarsys/data_objects/email_spec.rb]: omg! yes! i'm guilty :(

giphy-downsized 29

**Comment by kangguru** [comment]
@mungler its fixed 🍾

**Comment by mungler** [comment]
Great stuff. Merry Xmas!

**Comment by kangguru** [comment]
thanks a lot! 🎆 

will you find some time to make a new release?

**Pull request merged**